geocoding/example/lib/plugin_example/geocode_page.dart

```diff
@@ -105,7 +105,7 @@ class _GeocodeWidgetState extends State<GeocodeWidget> {
                   onPressed: () {
                     locationFromAddress(_addressController.text)
                         .then((locations) {
-                      final output = locations[0].toString();
+                      final output = locations![0].toString();
 
                       print(locations[0].toString());
```

Comment:
Since the native implementation can return `null` when no results are found I think we should handle this better here. Maybe we can replace this with:

```suggestion
                      var output = 'No results found.';
                      if (locations != null && locations.isNotEmpty()) {
                        output = locations[0].toString();
                      } 
```

We can remove the `print` statement all together.